open_mastr/utils/config.py

```diff
@@ -222,6 +222,11 @@ def _filenames_generator():
             {loc: f"{prefix}_{loc}_raw.csv" for loc in API_LOCATION_TYPES}
         )
 
+        # Add filenames for additional tables
+        filenames["raw"].update({"additional_table":
```

Comment:
Yes, that's right. I thought about this but wanted to tackle it in another PR before the next release as this one is really full already.
A simple solution would be to remove the `if not` here https://github.com/OpenEnergyPlatform/open-MaStR/blob/46f51056636404e2c706ab9eb69bf164083f4d54/open_mastr/utils/config.py#L143-L151
However, this would lead to an update of the filenames.yml everytime the Mastr class is called and longer runtimes, as the function is called in `setup_project_home()` which is called in `__init__.py`.
Better solutions would check for e.g. the version or include refactoring. But both options should be handled in another issue IMO.

Reply:
I created a new issue #410. Your suggestion makes sense for me.